fastlane/Deliverfile

```diff
@@ -0,0 +1,10 @@
+###################### More Options ######################
+# If you want to have even more control, check out the documentation
+# https://github.com/fastlane/fastlane/blob/master/deliver/Deliverfile.md
+
+
+###################### Automatically generated ######################
+# Feel free to remove the following line if you use fastlane (which you should)
+
+app_identifier "de.tum.campusapp" # The bundle identifier of your app
```

Comment:
That's the configuration file for fastlane deliver. "de.tum.campusapp" is the bundle id we use at the published app if I'm right.

However, this file probably shouldn't be tracked in git, since it's personalized. 

I think you have also the permission to add builds to iTunes Connect, right? Then you can initiate deliver by running "fastlane deliver" in the project root once. 

We can keep track of the changelog, screenshots, store description etc. in the repo in the _metadata_ (I think?) directory. We can then use fastlane deliver to publish those metadata to iTunes Connect or even submit new builds for review.

See https://github.com/fastlane/fastlane/tree/master/deliver for usage details.